portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Hello world!','你好，世界！'];
```

Comment:
nit: add space after comma in lists.

In case you aren't familiar, "nit" in code reviews is usually a small change that doesn't really matter, but is best for good code, like code formatting, naming, etc.